### PR TITLE
TCH-27 - update api document for path from user to a store

### PR DESCRIPTION
**Pull request by imwillim**

### CODE REVIEW

File: API.apib [hunk 1]
```diff
@@ -234,3 +234,30 @@ Response 200 (application/json)
             ]
      }
   }
+
+# Get path from user location to specific store
+GET api/v1/tch/stores/:store_id/path{?address}
+ Parameters:
+     + store_id (UUID)
+     + address (string)
+ Response 200 (application/json)
```
Comment (bmhuyquoc104) [inline API.apib:243]: how about response when request fails

File: API.apib [hunk 2]
```diff
@@ -234,3 +234,30 @@ Response 200 (application/json)
             ]
      }
   }
+
+# Get path from user location to specific store
+GET api/v1/tch/stores/:store_id/path{?address}
```
Comment (bmhuyquoc104) [inline API.apib:239]: should change to direction, can we have options for transportation

Comment (bmhuyquoc104) [inline API.apib:239]: why store id not in the {} here

Comment (bmhuyquoc104) [inline API.apib:239]: how do you plan to extract the address or extract correct info from this address, for example if I type
1012 đường 1 phường 3 quận 10
or 
1012duong1phuong3quan10

File: API.apib [hunk 3]
```diff
@@ -234,3 +234,30 @@ Response 200 (application/json)
...
+ Parameters:
+     + store_id (UUID)
```
Comment (bmhuyquoc104) [inline API.apib:241]: should add optional or required for each param, to ensure FE validation for correct one

**Review by bmhuyquoc104 — CHANGES_REQUESTED** [review]

File: API.apib (hunk 2 above)
Comment (imwillim) [reply API.apib:239]: I think using path variable is better than query parameter for `store_id` since a store does not often change address

Comment (imwillim) [reply API.apib:239]: Options for transportation are `driving`, `cycling` & `walking`. I thought that we agreed to use only `driving`

Comment (imwillim) [reply API.apib:239]: In Mapbox API, we are able to type `1012 đường 1 phường 3 quận 10`. However, using only `phường 3 quận 10` makes navigation api work better

Comment (bmhuyquoc104) [reply API.apib:239]: so how to extract that info :) if user enter something like this 
1012duong1phuong3quan10, we never know what user gonna input

Comment (bmhuyquoc104) [reply API.apib:239]: yeah, currently only support driving but however in future we need to expand it

File: API.apib [hunk 4]
```diff
@@ -234,3 +234,42 @@ Response 200 (application/json)
...
+  }
+
+  Response 422 (application/json)
+  {
+      message: 'Longitude must be between -180 and 180'
+  }
+
+  Response 400 (application/json)
+  {
+      message: 'Request failed after 3 attempts'
```
Comment (bmhuyquoc104) [inline API.apib:274]: user don't understand what 3 attemps mean here, just simply returns server error or server not avaiable

**Review by bmhuyquoc104 — APPROVED** [review]

**Pull request merged**